docs/documentation/writing-documentation.md

```diff
@@ -148,6 +155,58 @@ $ git push upstream new_docs:itb.new_docs
     Since there is only one ITB docs area, simultaneous new commits to different `itb.*` branches will overwrite each other's changes. To re-deploy your changes, find your [Travis-CI build](https://travis-ci.org/opensciencegrid/docs/branches) and restart it **BUT** coordinate with the author of the other commits to avoid conflicts.
 
 
+Upgrading to MkDocs v1
+----------------------
+
+The version of MkDocs used to generate documentation pages is determined by the submodule of `osg-ci-scripts`
+found in the `ci` directory.
+Most current documentation areas use MkDocs 0.17.5.
+However, we are planning to upgrade each repo to use MkDocs 1.0.4 or newer,
+which contains additional functionality and bug fixes.
+
+Before upgrading, you must fix the following incompatibilities:
+
+- Change the `pages:` section of `mkdocs.yml` to a `nav:` section.
+  The section contents are identical; only the name is changing.
+
+- Update all of the links in the documents as follows:
+  - Ensure links do not end in `.md`
+  - Ensure links are document-relative, not site-relative (e.g. `../software/development-process`
+    instead of `/software/development-process`
+
+Once these are done, use the following procedure to update:
+
+1. Create an itb branch from master, e.g.
```

Comment:
We should have a step or sentence here indicating that this needs to be done from within the repo they're updating.